vimc/montagu-webapps
# VIMC-4809 Add guidance pages for touchstone 202108test

To test:
- Modify `default.ts` so that `is2021Touchstone` returns `true` for a touchstone existing on your system e.g. `op-2018-1` (or maybe create a `202108test` touchstone, though I couldn't do that for some reason)
- Build/run contrib app
- Visit http://localhost:5000/contribution/IC-Garske/responsibilities/op-2018-1/
- Contents of "Useful links" should look like documents attached VIMC-4809. Note that help links won't work locally without the relevant resources but do work on UAT/science/prod.

Notes:
- This guidance is displayed only for the `202108test` touchstone, rather than any touchstone beginning with `202108` (or just `2021`) because the text refers explicitly to practice runs - we expect further guidance to be issued for the real ones.
- I've assumed links to Montagu reports provided in documents will be accessible to relevant users. If not we could convert reports to PDF and attach.
